pkg/testutils/certificate.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2019 The Jaeger Authors.
```

Comment:
It doesn't feel like we should have this code (which is also hard to test). Go already comes with `crypto/tls/generate_cert.go` that we can run (there's also installable version https://github.com/odeke-em/ssg).

Reply:
sure, removed that part of the PR